### ISSUE TITLE
Is it possible for ME Security Terminal to come back

### ISSUE THREAD

**Issue by Sichongzou** [AppliedEnergistics/Applied-Energistics-2#7564]
hello

I noticed that MOD removed ME Security Terminal in version 1.20.1

What I want to ask is, how can I ensure the security of my data network in the current version, only with the help of various protection plugins on the server?

I really believe that the functions implemented by ME Security Terminal are very compatible with the characteristics of MOD. Although it may appear less secure in other MODs, it is really regrettable that it was deleted

**Comment by Sichongzou** [comment]
But the actual effect of ME Security Terminal is not simply to prevent destruction

He can still control who can take things from my AE

I think this is a very cool gameplay in the server

With the existence of ME Security Terminal

I can use Quantum Network Bridge as I please to expand my AE range

In the past, maybe all I needed was to protect the output of Quantum Network Bridge with plugins

On the other hand, I don't need to protect myself because with ME Security Terminal, he can't retrieve items from my network.

But now I have to

**Comment by Sichongzou** [comment]
Yes, even with the presence of secure terminals, if the drive is damaged by other MODs, the disk will still fall out. But in my application case, it refers to the situation where the driver is protected by a plugin, and then the channel is transmitted to a distance through a quantum ring. The input quantum ring is protected by other plugins, while the output quantum ring is not protected.


In this case, even if the output quantum ring is forced to place a terminal on my quantum ring through other MODs, the other party cannot remove any items from my network because my entire network is protected by a secure terminal.

In this way, as long as my main network, which is the input end of the quantum ring, is protected by other plugins, I can freely build a quantum network bridge anywhere. Don't worry about the item being taken away because the other side of the quantum bridge is not protected

Perhaps my description is a bit strange, it's because I used translation software. If there is any semantic ambiguity, please let me know and I will try my best to explain it

**Comment by Mujaca** [comment]
Wouldn't it be smarter to bring back the terminal with improved code? Since many people, especially on public servers, used it to protect their ME. I know many people, that didn't know it was possible to just break the security terminal to gain access to the system. 
And as a software developer, I don't see how removing a feature is a bugfix for that feature.
